package.json

```diff
@@ -12,6 +12,18 @@
   "activationEvents": [],
   "main": "./out/extension.js",
   "contributes": {
+	"configuration":[
+		{
+			"title": "Analytics Settings",
+			"properties": {
+				"jetpackVSCode.enableAnalytics": {
+					"type": "boolean",
+					"default": true,
```

Comment:
Good call! I wonder if there's some kind of pop up we can add when it's installed so people can opt in 🤔 Otherwise I can't imagine most people will find and turn on the setting on their own (I know I certainly wouldn't 😅 )

264e53f9f83bfcd80e33cb1c1cbae6e83ef86d68

Reply:
We can maybe sneak that into our repo `.vscode` folder. We can also show a popup the first time you run a command, but either way it's outside of the scope for this PR I think.